samples/csharp/MicrosoftGraphSDK/Assignment.cs

```diff
@@ -36,6 +36,31 @@ public class Assignment
             }
         }
 
+        /// <summary>
+        /// Get all the assignments from the class
+        /// </summary>
+        /// <param name="client"></param>
+        /// <param name="classId"></param>
+        /// <returns>IEducationClassAssignmentsCollectionPage</returns>
+        public static async Task<IEducationClassAssignmentsCollectionPage> GetAssignments(
```

Comment:
add suffix "async" to all async methods e.g. GetAssignments to GetAssignmentsAsync